src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -67,19 +67,6 @@ public void InvokeRecoveryError(object sender, string message, Exception e)
             handler(this, new RecoveryErrorEventArgs(message, e));
         }
 
-        public void InvokeNonDurabaleFileSystemError(object sender, string message, Exception e)
-        {
-            var handler = OnNonDurabaleFileSystemError;
-            if (handler == null)
-            {
-                throw new InvalidDataException(message + Environment.NewLine +
```

Comment:
No, this needs to remain. If we aren't hooked up, this should throw.
We handle that at the ravendb level, but Voron should enforce that if you are running in unsafe mode, you do so explicitly.